py/desiutil/test/test_bitmask.py

```diff
@@ -6,8 +6,9 @@
                         print_function, unicode_literals)
 # The line above will help with 2to3 support.
 import unittest
-from ..bitmask import BitMask, _MaskBit
+from desiutil.bitmask import BitMask, _MaskBit
```

Comment:
With the original relative import syntax, you cannot run
```
[desiutil] $ python py/desiutil/test/test_bitmask.py
Traceback (most recent call last):
  File "py/desiutil/test/test_bitmask.py", line 9, in <module>
    from ..bitmask import BitMask, _MaskBit
ValueError: Attempted relative import in non-package
```
With the old-style fully qualified package path syntax, you can:
```
[desiutil] $ python py/desiutil/test/test_bitmask.py
.......
----------------------------------------------------------------------
Ran 7 tests in 0.011s

OK
```
This is a handy trick when debugging a single failing test since you don't have to keep rerunning every passing test via `python setup.py test`.  I always re-run that at the end to make sure I didn't accidentally break something else, but being able to run just a single test file is useful.  This is even more useful for packages like desisim where the full suite of tests can take a few minutes to run.

If the old-style syntax actually creates a problem in this case, let me know.